app/models/site_config.rb

```diff
@@ -29,6 +29,7 @@ class SiteConfig < RailsSettings::Base
   field :main_social_image, type: :string, default: "https://thepracticaldev.s3.amazonaws.com/i/6hqmcjaxbgbon8ydw93z.png"
   field :favicon_url, type: :string, default: "favicon.ico"
   field :logo_svg, type: :string, default: ""
+  field :primary_sticker_image_url, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/rainbowdev.svg"
```

Comment:
Just want to make sure this is the url we want to be default.

Reply:
I think it's fine; thoughts @benhalpern?